RELEASE_NOTES.md

```diff
@@ -3,6 +3,19 @@ MSA Release Notes
 
 ### Next
 
```

Comment:
Isn't there supposed to be a version number here, with a `View Diff` link underneath?

Reply:
I believe this will added by the eIDAS team when they actually generate the release. We only needed to add our bits to this section ready for them.